### PR TITLE
EOF follows posix/unix standard

### LINKED ISSUE #90: Editor config requires no newline at end of file
different issue: the editor config requires no newline at end of file, editing the file via github web always adds newline at end of file

_Originally posted by @larshp in https://github.com/SAP/abap-file-formats/issues/87#issuecomment-839654258_

### PR DESCRIPTION
**Pull request by albertmink**
closes #90

### CODE REVIEW

File: file-formats/intf/zif_intf_v1.intf.abap [hunk 1]
```diff
@@ -0,0 +1,29 @@
+INTERFACE zif_intf_v1 PUBLIC.
+
+  TYPES: BEGIN OF header,
```
Comment (larshp) [inline file-formats/intf/zif_intf_v1.intf.abap:3]: it looks like this is an old file that made it into this branch? filename in main is `zif_aff_intf_v1.intf.abap`

Comment (albertmink) [reply file-formats/intf/zif_intf_v1.intf.abap]: already started to be worried. Yep, this file does no longer belong to the repo

**Review by larshp — APPROVED** [review]
LGTM 👍

**Review by schneidermic0 — APPROVED** [review]
Looks good to me

**Pull request merged**